packages/editor/src/client/blocks/richTextBlock/toolbar/buttons.tsx

```diff
@@ -50,6 +51,18 @@ export function FormatButton({format, children}: FormatBlockButtonProps) {
   )
 }
 
+export function FormatFontColor() {
+  const editor = useSlate()
+
+  return (
+    <ToolbarButton>
```

Comment:
Can you try putting this in a submenu consistent with table, links, and emojis?

Reply:
I agree. 👍🏾

Reply:
@penk85 @roosnic1 Changed :+1: 
![Screenshot from 2021-04-06 13-47-30](https://user-images.githubusercontent.com/69328362/113731737-4bf06180-96f9-11eb-854b-3a89ca1cbbce.png)